workflow-core/README.md

```diff
@@ -9,6 +9,14 @@ are `jvm`, `ios`, and `iosSimulatorSimulatorArm64`. If you are having issues wit
 ensure you have the correct version of XCode installed and can launch a simulator as it's specified
 in the gradle build file (Currently iPhone 14).
 
+You can also choose to specify your targets for build and test with the property `workflow.targets`
+as either `kmp`, `android`, `ios`, `js`. The default is `kmp` (all the targets). Set this in your
+gradle.properties or specify the property in your gradle command, e.g.:
```

Comment:
> Set this in your gradle.properties

It's probably worth calling out that this would be `~/.gradle/gradle.properties`, and not the project properties.

Reply:
Done.